### PR TITLE
Add funding information to website

### PR DESCRIPTION
**Pull request by jennaswa**
Fixes https://github.com/MDAnalysis/MDAnalysis.github.io/issues/299 and addresses conversations related to transparency about who is paid by grants as it relates to roles and responsibilities

Key personnel across grants (@fiona-naughton, @IAlibay, @lilyminium, @micaela-matta, @orbeckst, @richardjgowers) should please confirm their grant information is represented properly. @orbeckst - I especially wasn't sure how to best represent the NSF funding here.

Since many of the SDGs were awarded to MDAnalysis before it became a NumFOCUS sponsored project, I do not have access to historical records about key personnel involved on these grants. Please add if you know any of this information.

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -152,42 +152,90 @@ tax adviser or the IRS about your particular tax situation.
...
-    sponsored project" alt="NumFOCUS Sponsored" style="display:
-    inline; float: left; height: 4em; margin: 0 0.5em" /></a>
+### [Chan Zuckerberg Initiative][] (CZI)
+MDAnalysis has been supported by the [Essential Open Source for Science](https://chanzuckerberg.com/rfa/essential-open-source-software-for-science/) (EOSS) program from the CZI DAF, an advised fund of Silicon Valley Community Foundation (funder DOI 10.13039/100014989)
 
+- EOSS5, 2022-253062 (**2022**): [MDAnalysis: Outreach and Project Manager](https://chanzuckerberg.com/eoss/proposals/mdanalysis-outreach-and-project-manager/) (**Key Personnel**: @IAlibay, @micaela-matta, @orbeckst, @richardjgowers (*PI*); **Hired Personnel**: @jennaswa)
+- EOSS4, DAF2021-237663, DOI [https://doi.org/10.37921/426590wiobus](https://doi.org/10.37921/426590wiobus) (**2021**): [MDAnalysis: Faster, Extensible Molecular Analysis for Reproducible Science](https://chanzuckerberg.com/eoss/proposals/mdanalysis-faster-extensible-molecular-analysis-for-reproducible-science/) (**Key Personnel**: @fiona-naughton, @IAlibay, @lilyminium, @orbeckst (*PI*), @richardjgowers; **Hired Personnel**: @ianmkenney)
```
Comment (IAlibay) [inline about.md:161]: @hmacdope is missing somewhere here - not sure if key

**Review by IAlibay — COMMENTED** [review]
Thanks so much for this @jennaswa - I have a quick question on "key personnel" which is more me overthinking so I'll ping you on discord.

File: about.md (hunk 1 above)
Comment (jennaswa) [reply about.md]: Added Hugo to EOSS4

**Review by lilyminium — APPROVED** [review]
LGTM -- thanks @jennaswa!

**Comment by orbeckst** [comment]
I will have a look after Thanksgiving holiday. Thanks for all the work!Am 11/23/23 um 04:08 schrieb Jenna M Swarthout Goddard ***@***.***>:﻿
@jennaswa requested your review on: #339 Add funding information to website.

**Review by hmacdope — APPROVED** [review]
LGTM

**Comment by RMeli** [comment]
@IAlibay, has your comment been clarified/discusses?

**Comment by jennaswa** [comment]
If there are no additional comments/suggestions/requested changes, I will plan to merge this PR on Friday, Jan 12.

File: about.md [hunk 2]
```diff
@@ -152,42 +152,90 @@ tax adviser or the IRS about your particular tax situation.
...
 
+<a href="https://developers.google.com/season-of-docs"> <img
+    src="https://developers.google.com/season-of-docs/images/SeasonofDocs_Icon_Grey_300ppi_trimmed.png"
+    title="Google Season of Docs" alt="Google Season of Docs"
+    style="display: inline; float: left; height: 4em; margin: 0 0.5em" /></a>
+
+### [National Science Foundation](https://www.nsf.gov/)
+Earlier work was partially supported by the NSF (as part of award ACI-1443054).
+
+- NSF DIBBS award, ACI-1443054 (**2014**): [CIF21 DIBBs: Middleware and High Performance Analytics Libraries for Scalable Data Science](https://www.nsf.gov/awardsearch/showAward?AWD_ID=1443054) (**MDAnalysis Personnel**: @orbeckst)
```
Comment (orbeckst) [inline about.md:198]: REU students 
- Ian Kenney @ianmkenney 
- Edis Jakupovic @edisj
- Henry Mull @hfmull
- Robert Delgado @rbrtdlgd
- Alia Lescoulie @Alescoulie
- Kathleen Clark @kaceyaurum

**Review by orbeckst — APPROVED** [review]
Looking good, but could you please explicitly include the NSF REU students who contributed to MDA, see comment? Just handles as for everybody else, I just wrote the names as this is as I had the information. Thanks.

File: about.md (hunk 2 above)
Comment (jennaswa) [reply about.md:198]: Added - please ensure the years match up with the information you have.

File: about.md [hunk 3]
```diff
@@ -152,42 +152,99 @@ tax adviser or the IRS about your particular tax situation.
...
+
+The following students were sponsored to work on MDAnalysis through the [NSF Research Experience for Undergraduates](https://www.nsf.gov/crssprgm/reu/) (REU) program.
+
+- **2021**: @ALescoulie
+- **2020**: @edisj
```
Comment (orbeckst) [inline about.md:203]: I forgot Nik Awtrey's work on PMDA and Edis was also REU in 2021.
```suggestion
- **2021**: @ALescoulie, @edisj
- **2020**: @edisj
- **2019**: @nawtrey
```

**Review by orbeckst — APPROVED** [review]
Thanks @jennaswa !

(I am making an update to the REU, given that we list by year.)

**Pull request merged**